routes/api.php

```diff
@@ -147,11 +147,11 @@
         Route::get('/users/me/checkins', 'Api\Place\PlaceCheckinController@getUserCheckInCollection')
             ->name('user.me.getUserCheckins');
 
-        Route::post('/user/{id}/follow', 'Api\User\UserFollowsController@followUser');
-        Route::post('/user/{id}/unfollow', 'Api\User\UserFollowsController@unfollowUser');
+        Route::post('/users/{id}/follow', 'Api\User\UserFollowsController@followUser');
```

Comment:
Use next endpoints:
POST /users/{id}/followers
DELETE /users/{id}/followers

or 'followed'. The reason is no verbs in the endpoints, the plural noun is best, but in this case difficult with meaning